testsuite/features/reposync/srv_sync_repo_sync.feature

```diff
@@ -0,0 +1,55 @@
+# Copyright (c) 2023 SUSE LLC
+# Licensed under the terms of the MIT license.
+
+Feature: Strict Mode for Channel Synchronization
+  In order to control strict mode synchronization
+  As an admin user
+  I want to be able to toggle strict mode synchronization via the UI and command line
+
+Scenario: Create Test-strict-channel
+  Given I am authorized for the "Admin" section
+    And I prepare a channel clone for strict mode testing
+    And I follow the left menu "Software > Manage > Channels"
+    And I follow "Create Channel"
+    And I enter "Test-Strict-Channel" as "Channel Name"
+    And I enter "test-strict-channel" as "Channel Label"
+    And I enter "Test-Strict-Channel for testing" as "Channel Summary"
+    And I click on "Create Channel"
+    Then I should see a "Channel Test-Strict-Channel created." text
+
+Scenario: Prepare repos for strict test
+    Given I am authorized for the "Admin" section
+    And I follow the left menu "Software > Manage > Repositories"
+    And I follow "Create Repository"
+    And I enter "fake-rpm-repo-modified" as "label"
+    And I enter "http://localhost/pub/TestRepoRpmUpdates_STRICT_TEST/" as "url"
+    And I uncheck "metadataSigned"
+    And I click on "Create Repository"
+    Then I should see a "Repository created successfully" text
+
+@rrichardson-test
+Scenario: Testing strict mode
+    Given I am authorized for the "Admin" section
+    And I follow the left menu "Software > Manage > Channels"
+    And I follow "Test-Strict-Channel"
+    And I follow "Repositories" in the "content area"
+    And I check "fake-rpm-repo" in the list
+    And I click on "Save Repositories"
+    And I follow "Sync" in the "content area"
+    And I click on "Sync Now"	
+    And I wait at most 60 seconds until I do not see "Repository sync is running." text, refreshing the page
+    And I check the amount of packages in channel "test-strict-channel"
```

Comment:
This step is not check (verifying/asserting) anything. It is storing and printing.

Reply:
renamed step to `I store the amount...`